src/NServiceBus.RavenDB/Outbox/RavenDBOutboxExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static EndpointConfiguration SetTimeToKeepDeduplicationData(this Endpoint
             var now = DateTime.UtcNow;
             if (now - timeToKeepDeduplicationData >= now)
             {
-                throw new ArgumentException("Specify a non-negative TimeSpan. The cleanup process removes entries older than the specified time to keep deduplication data, therefore the time span cannot be negative.", "timeToKeepDeduplicationData");
+                throw new ArgumentException("Specify a non-negative TimeSpan. The cleanup process removes entries older than the specified time to keep deduplication data, therefore the time span cannot be negative.", nameof(timeToKeepDeduplicationData));
```

Comment:
Shouldn't we mention the API here the customer should use? Shouldn't this be validated in that extension method?